views/index.jade

```diff
@@ -25,17 +25,17 @@ block content
         div.col-md-6.whatis
           h2 Build unstoppable applications
 
-          p Ethereum is a
+          p Ethereum is a &nbsp;
```

Comment:
actually the non breaking space character is unnecessary. A simple space before the end of the line should be enough